### PR TITLE
Made emoji clarifications and added a few new emojis

### PR DESCRIPTION
**Pull request by kevintongg**
Let's see if this works

### CODE REVIEW

File: src/commands/weather.js [hunk 1]
```diff
@@ -74,23 +77,25 @@ class Weather extends Aquarius.Command {
...
+        return EMOJI_LIST.HURRICANE;
       case '3':    // Severe Thunderstorms
       case '4':    // Thunderstorms
       case '37':   // Isolated Thunderstorms
       case '38':   // Scattered Thunderstorms
       case '39':   // Scattered Thunderstorms
       case '45':   // Thundershowers
-      case '47':   // Isolated Thundershowers
         return EMOJI_LIST.THUNDER;
+      case '47':   // Isolated Thundershowers
```
Comment (IanMitchell) [inline src/commands/weather.js]: Was this moved intentionally?

File: src/commands/weather.js [hunk 2]
```diff
@@ -112,8 +117,8 @@ class Weather extends Aquarius.Command {
       case '19':   // Dust
       case '20':   // Foggy
       case '21':   // Haze
-      case '22':   // Smoky
         return EMOJI_LIST.FOGGY;
+      case '22':   // Smoky
```
Comment (IanMitchell) [inline src/commands/weather.js]: Same here

File: src/commands/weather.js [hunk 3]
```diff
@@ -9,15 +9,18 @@ const EMOJI_LIST = {
   MIXED: '🌦',
   OVERCAST: '☁',
   THUNDER: '⛈',
+  SNOW: '❄️',
   SNOWY: '🌨',
   TORNADO: '🌪',
-  FOGGY: '🌁',
+  HURRICANE: '🌀',
+  FOGGY: '🌫',
```
Comment (IanMitchell) [inline src/commands/weather.js]: Checks out on iOS; @MisterFudd can you check on Android?

File: src/commands/weather.js (hunk 1 above)
Comment (kevintongg) [inline src/commands/weather.js]: Yes, because the emoji does not really symbolize isolated thundershowers as it does to where I moved it to.

Comment (IanMitchell) [inline src/commands/weather.js]: Does the snow emoji really indicate that though?

Comment (kevintongg) [inline src/commands/weather.js]: If I look it again now, maybe not. But does it need to have clouds to symbolize snow?

File: src/commands/weather.js (hunk 2 above)
Comment (IanMitchell) [inline src/commands/weather.js]: I think for Smoky the foggy emoji makes more sense than the windy one - swap this and I'll merge

**Pull request merged**